packages/core/src/utils.ts

```diff
@@ -619,36 +619,39 @@ export const readBuildInfo = (buildInfoPath: string): BuildInfo => {
     fs.readFileSync(buildInfoPath, 'utf8')
   )
 
-  validateBuildInfo(buildInfo)
-
   return buildInfo
 }
 
-export const validateBuildInfo = (buildInfo: BuildInfo): void => {
+export const validateBuildInfo = (
+  buildInfo: BuildInfo,
+  integration: Integration
+): void => {
   if (!semver.satisfies(buildInfo.solcVersion, '>0.5.x <0.9.x')) {
     throw new Error(
       `Storage layout for Solidity version ${buildInfo.solcVersion} not yet supported. Sorry!`
     )
   }
 
-  if (
-    !buildInfo.input.settings.outputSelection['*']['*'].includes(
-      'storageLayout'
-    )
-  ) {
-    throw new Error(
-      `Did you forget to set the "storageLayout" compiler option in your Hardhat/Foundry config file?`
-    )
-  }
+  if (integration === 'hardhat') {
+    if (
+      !buildInfo.input.settings.outputSelection['*']['*'].includes(
+        'storageLayout'
+      )
```

Comment:
This check fails during partial rebuilds in Foundry b/c for some reason foundry outputs the settings on a per-contract basis so we have to do this check separately. That being said, there doesn't seem to be any underlying issue that makes the required info unavailable to us.

Reply:
This isn't really an issue in practice b/c foundry will automatically do a full rebuild if you run a script after changing your config.

Reply:
Ah nice, could you just document this in the relevant section of `get-bundle-info` then?